ir/src/main/scala/org/scalajs/ir/ScalaJSVersions.scala

```diff
@@ -57,8 +57,7 @@ class VersionChecks private[ir] (
    *    versions. Therefore, the full [[binaryEmitted]] version is used.
    */
   final val binaryCross: String = {
-    val needsFull =
-      binaryPreRelease.fold(false)(_ != "SNAPSHOT" || binaryMinor == 0)
+    val needsFull = binaryPreRelease.isDefined && binaryMinor == 0
```

Comment:
The comment on this method needs to be updated.

Reply:
Indeed. Updated.